waspc/data/Generator/templates/react-app/src/auth/useAuth.ts

```diff
@@ -1,12 +1,14 @@
 import { useQuery } from '../queries'
 import api, { handleApiError } from '../api'
 import { HttpMethod } from '../types'
+// todo(filip): turn into a proper import
+import { type SanitizedUser } from '../../../server/src/_types/' 
 
 export default function useAuth(queryFnArgs, config) {
```

Comment:
I've decided not to go with the module boundaries principle (https://github.com/wasp-lang/vscode-wasp/pull/6#discussion_r970774084) to avoid unnecessary verbosity.

This short function that's basically a wrapper to two other (explicitly typed) public functions: `useQuery` and `getMe`. If that changes, we'll make the type signature explicit.